From ee83e7f1acc80de721d70b7d03f8cb43d2f73c19 Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@cloud.ionos.com>
Date: Tue, 22 Jan 2019 13:22:20 +0100
Subject: [PATCH rdma-core] pyverbs: Add shebang to ib_devices.py example

The example file ib_devices.py has the execution bit set, but it misses
the shebang.

Signed-off-by: Benjamin Drung <benjamin.drung@cloud.ionos.com>
---
 pyverbs/examples/ib_devices.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyverbs/examples/ib_devices.py b/pyverbs/examples/ib_devices.py
index c38164b4..28884376 100755
--- a/pyverbs/examples/ib_devices.py
+++ b/pyverbs/examples/ib_devices.py
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
 # SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
 # Copyright (c) 2018, Mellanox Technologies. All rights reserved. See COPYING file
 
-- 
2.17.1

